roles/update_etc_hosts/README.md

```diff
@@ -0,0 +1,66 @@
+Role Name
+=========
+
+This role can be used to reliably update the /etc/hosts file.
+
+<!---
+Requirements
+------------
+
+ Any pre-requisites that may not be covered by Ansible itself or the role should be mentioned here. For instance, if the role uses the EC2 module, it may be a good idea to mention in this section that the boto package is required.
+--->
+
+Role Variables
+--------------
+
+This role needs a a dictonary `update_etc_hosts_list` which contains the parameters for the hostfile.
+It can be used for adding and deleting entries from `/etc/hosts`
+
+<!---
+Dependencies
+------------
+
+A list of other roles hosted on Galaxy should go here, plus any details in regards to parameters that may need to be set for other roles, or variables that are used from other roles.
+--->
+
+Example Playbook
+----------------
+
+If you want to setup/add entries your etc hosts you can use this snippet
+
+```[yaml]
+- name: Ensure /etc/hosts is updated
+  include_role: sap_update_etc_hosts
+  var:
+        update_etc_hosts_list:
+                - node_ip: 1.2.3.4
+                  node_name: host1
+                  aliases:
+                    - alias1
+                    - anotheralias2
+                  comment: "Here comes text after hashsign" (defaults to hana_site)
```

Comment:
The example mentions "comment" but the `meta/argument_specs.yml` file has `node_comment`.

Reply:
fixed